pkg/api/routes.go

```diff
@@ -149,5 +155,11 @@ func (h *Handler) GetRoutes() []Route {
 			"/log",
 			h.SetLogLevel,
 		},
+		{
+			"GetMetrics",
```

Comment:
i was thinking of having metrics be a sub-resource for every resource because it would map better when we do TPR for every rook resources.